tests/filecheck/backend/csl/print_csl.mlir

```diff
@@ -389,6 +403,14 @@ csl.func @builtins() {
 // CHECK-NEXT:   return;
 // CHECK-NEXT: }
 // CHECK-NEXT:
+// CHECK-NEXT: fn constants() void {
+// CHECK-NEXT:   var v0 : [const27]i16 = @constants(const27, const27);
+// CHECK-NEXT:   const v1 : [const27]i16 = @constants(const27, const27);
+// CHECK-NEXT:   const v2 : [const27]i32 = @constants(const27, 100);
```

Comment:
Shouldn't this be `@constants([const27]i6, const27)`?

Reply:
dayyyum, you are right! good catch!